src/js/components/accordion.js

```diff
@@ -70,7 +70,7 @@ Accordion.prototype.show = function ($button) {
  */
 Accordion.prototype.hideAll = function () {
   var self = this;
-  this.$('button').each(function () {
+  this.$('ul > li > button, .usa-accordion-button').each(function () {
```

Comment:
Why not just use the `.usa-accordion-button` as the target? Totally open if you think we need to cover both use cases, but seems like the class specificity will suffice.